pinot-common/src/main/java/org/apache/pinot/common/metrics/ServerMeter.java

```diff
@@ -62,6 +62,7 @@
   RELOAD_FAILURES("segments", false),
   REFRESH_FAILURES("segments", false),
   UNTAR_FAILURES("segments", false),
+  RETRY_EXCEEDED_FAILURES("segments", false),
```

Comment:
can you pick a better name

Reply:
Renamed to `DOWNLOAD_SEGMENT_RETRY_EXCEEDED_FAILURES`. Feel free to suggest any better name. 😉

Reply:
How about `SEGMENT_DOWNLOAD_FAILURE`?